src/bin/release-please.ts

```diff
@@ -1,206 +1,13 @@
 #!/usr/bin/env node
 
-// Copyright 2019 Google LLC
```

Comment:
Need to keep the license header.

Reply:
This has been updated. I still seem to be hitting an error in CI for a missing license header in `__snapshots__/runner.js` which is expected to be ignored...